projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/services/product-facet.service.ts

```diff
@@ -24,6 +24,9 @@ export class ProductFacetService {
     protected productListComponentService: ProductListComponentService
   ) {}
 
+  model$: Observable<ProductSearchPage> =
+    this.productListComponentService.model$;
+
```

Comment:
This issue is caused by `filterForPage`.
line 90 should be `return page.freeTextSearch === state.params.query.split(':')[0];`